<launch>
    <group ns="indy7">
        <node name="task_controller" pkg="controll_robots" type="TaskController_indy7.py" output="screen"/>
        <node name="joint_controller" pkg="controll_robots" type="JointController_indy7.py" output="screen"/>
        <node name="cmd_talker" pkg="controll_robots" type="Controller_cmd.py" output="screen"/>
    </group>

    <group ns="rb10">
        <node name="task_controller" pkg="controll_robots" type="TaskController_rb10.py" output="screen"/>
        <node name="joint_controller" pkg="controll_robots" type="JointController_rb10.py" output="screen"/>
        <node name="cmd_talker" pkg="controll_robots" type="Controller_cmd.py" output="screen"/>
    </group>

    <!-- <node name="controller_ui" pkg="controll_robots" type="test_UI.py" output="screen"/> -->
    <!-- <node name="combine_tf" pkg="combine_tf" type="tf_manager.py" output="screen"/>  -->
</launch>
